huaweicloud/config/endpoints.go

```diff
@@ -74,6 +74,13 @@ var allServiceCatalog = map[string]ServiceCatalog{
 		WithOutProjectID: true,
 		Product:          "IAM",
 	},
+	"identity_extension": {
```

Comment:
please rename the key and it's better to add this item after **identity**, also, you should add `identity_ext` in multiCatalogKeys at L23.
```suggestion
	"identity_ext": {
```